doc/operator-scope.md

```diff
@@ -121,7 +121,7 @@ This scope is ideal for the cases where an instance(CR) of some Kind(CRD) will b
         listKind: MemcachedList
         plural: memcacheds
         singular: memcached
-      scope: Namespaced
+      scope: Cluster
```

Comment:
By default, it will be Namespaced. So, just after following the steps the scope will be changed to `Cluster`.
In this way, at this moment it still Namespaced then I do not think that we should move forward with this PR. 
Is it make sense ?

Reply:
Hi Camila,

Thanks for your review!

I've read the doc once again and now I got it. Basically I only need to update ```pkg/apis/<group>/<version>/<kind>_types.go``` and then run ```operator-sdk generate crds```. 

Initially this statement in the doc ```For each CRD that needs to be cluster-scoped, update its manifest to be cluster-scoped.``` made me feel that I have to update ```deploy/crds/<full group>_<resource>_crd.yaml``` manifest manually. 

So your comment makes total sense and I'll close this PR.

Thanks! :)